great-idea-website/css/index.css

```diff
@@ -63,4 +63,118 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+  width: 80%;
+  min-width: 600px;
+  max-width: 1000px;
+  margin: 0 auto;
+  align-content: center;
+}
+.navbar{
+  margin: 0 auto;
+  justify-content: space-evenly;
+  word-spacing: 50px;
+  display:inline-block;
+  position: relative;
+  top: 1em;
+
+}
+.navbar nav a{
+	font-size: 10px;  
+	color: grey;
+	border: 1px transparent solid;
+	width: 80%;
+	height: 20%;
+	margin: 0 auto;	
+	padding-top: 50px;
+}
+.navbar .image1{
+  position: absolute;
+  right: -20em;
+  top: -.5em;
+ }
+.cont .slg{
```

Comment:
the css styles are fine, but the naming convention of the class names are difficult to understand.  Try to use something a bit more verbose.